runtime/executor/src/main/java/org/apache/nemo/runtime/executor/datatransfer/NemoEventDecoderFactory.java

```diff
@@ -84,6 +85,9 @@ public Object decode() throws IOException {
         final WatermarkWithIndex watermarkWithIndex =
           (WatermarkWithIndex) SerializationUtils.deserialize(inputStream);
         return watermarkWithIndex;
+      } else if (isWatermark == 0x02) {
```

Comment:
We need to change the variable name `isWatermark`

Reply:
done